plugins/lsp_python.lua

```diff
@@ -0,0 +1,40 @@
+-- mod-version:3
+
+local lspconfig = require "plugins.lsp.config"
+local common = require "core.common"
+local config = require "core.config"
+local core = require "core"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_python"
+local node_version = "node-v20.11.0"
+local installed_path_library = USERDIR .. PATHSEP .. "libraries" .. PATHSEP .. "nodejs"
+
+local server_path = installed_path_plugin .. PATHSEP .. "package" .. PATHSEP .. "langserver.index.js"
+
+local node_path
+if ARCH == "x86_64-linux" then
+  node_path = installed_path_library .. PATHSEP .. node_version .. "-linux-x64" .. PATHSEP .. "bin" .. PATHSEP .. "node"
+elseif ARCH == "x86_64-darwin" then
+  node_path = installed_path_library .. PATHSEP .. node_version .. "-darwin-x64" .. PATHSEP .. "bin" .. PATHSEP .. "node"
+elseif ARCH == "arm64-darwin" then
+  node_path = installed_path_library .. PATHSEP .. node_version .. "-darwin-arm64" .. PATHSEP .. "bin" .. PATHSEP .. "node"
+elseif ARCH == "x86_64-windows" then
+  node_path = installed_path_library .. PATHSEP .. node_version .. "-win-x64" .. PATHSEP .. "node.exe"
+end
```

Comment:
What do you think about adding a `nodejs.lua` file that returns the path?
This way plugins that depend on it can just require the file to get the current path, instead of having to update all of them in case it changes in the future.

Reply:
Good idea

Reply:
Should stay in this repo for now, as the `nodejs` library is in the manifest here.